EasyPost.Net/Utilities/JsonSerialization.cs

```diff
@@ -0,0 +1,105 @@
+using System;
```

Comment:
Serialization is hard and confusing - can we document with comments this code a bit to help future travelers know what it's all doing (at a high level)?

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)